avocado/core/loader.py

```diff
@@ -139,8 +139,6 @@ def _str_loaders():
         loaders = args.get('loaders', None)
         if not loaders:
             loaders = settings.get_value("plugins", "loaders", list, [])
-        if '?' in loaders:
```

Comment:
Oh yeah! But please also remove the mentions from `guides/user/chapters/loaders.rst`